.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: run-tests
+on: [pull_request]
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    strategy:
+        matrix:
+          python-version: [ 3.6, 3.7, 3.8 ]
+          include:
+            - os: ubuntu-latest
+              set_display: export DISPLAY=:99; Xvfb :99 -screen 0 1024x768x24 -ac -noreset & sleep 3
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .
+          pip install coveralls mock pytest-cov
+          sudo apt-get update
+          sudo apt-get -y -q install xvfb scrot zip curl libxml2-dev libxslt1-dev
```

Comment:
Lines 24 through 25 can be removed since you are not running any tests that require a screen